package.json

```diff
@@ -17,12 +17,15 @@
   },
   "dependencies": {
     "@chakra-ui/react": "^2.4.9",
+    "@emotion/react": "^11.11.4",
+    "@emotion/styled": "^11.11.5",
     "@tabler/icons-react": "^2.12.0",
     "@tailwindcss/typography": "^0.5.9",
     "@thedutchcoder/postcss-rem-to-px": "^0.0.2",
     "chakra-react-select": "^4.6.0",
     "date-fns": "^2.29.3",
     "date-fns-tz": "^2.0.0",
+    "framer-motion": "^11.0.24",
```

Comment:
I'm a little confused about those 3 packages, `@emotion/react`, `@emotion/styled`, and `framer-motion`.
Where use them and how?